drivers/xen/dom0/domctl.c

```diff
@@ -313,6 +313,22 @@ int xen_domctl_cacheflush(int domid,  struct xen_domctl_cacheflush *cacheflush)
 	return do_domctl(&domctl);
 }
 
+int xen_domctl_getvcpu(int domid, int vcpu, struct xen_domctl_getvcpuinfo *info)
```

Comment:
why are you using `int` types for `domid` and `vcpu`? They can't be negative.

Okay, I see that surrounding code uses `int` for `domid`, so you can leave it for consistency (but we will need to fix this in the future). But at least you can change type for `vcpu` to something more appropriate.

Reply:
Done